tests/unit/store/peek-record-test.js

```diff
@@ -41,3 +41,20 @@ test('peekRecord should return null if the record is not in the store ', functio
     assert.equal(null, store.peekRecord('person', 1), 'peekRecord returns null if the corresponding record is not in the store');
   });
 });
+
+test('peekRecord should assert if not passed both model name and id', function(assert) {
+  run(() => {
+    assert.expectAssertion(() => {
+      store.peekRecord('my-id');
+    }, /You need to pass both a model name and id/);
+  });
+});
+
+test('peekRecord should assert if passed a model class instead of model name', function(assert) {
```

Comment:
The appveyor tests appear to have passed, but appveyor reports a timeout. Might just need a kick to re-run the tests; afaict everything should be passing now.